README.md

```diff
@@ -226,6 +229,51 @@ The only required key/value in the signing_info dictionary is 'identity'.
 See the **SIGNED PACKAGES** section of the man page for `pkgbuild` or the **SIGNED PRODUCT ARCHIVES** section of the man page for `productbuild` for more information on the signing options.
 
 
+### Package notarization
+
+You may notarize **SIGNED PACKAGES** as part of the build process by adding a notarization dictionary to the build\_info.plist:
+
+```plist
+    <key>notarization</key>
```

Comment:
Let's make this "notarization_info" to match the style/format of "signing_info".

Reply:
Ok. I can change that.

Btw I just had an idea. Would it make sense to move notarization configuration under the `signing_info` since that is required for notarization to work or should we stick with separate dictionary object at root level of the `build_info`?

Reply:
Change from `notarization to `notarization_info`  is done in this fixup commit: edb1e02809d467b0cd580402829f987d042f9825